packages/components/tests/dummy/app/templates/components/dropdown.hbs

```diff
@@ -12,491 +12,675 @@
 
 <section>
   <h3 class="dummy-h3" id="section-component-api">Component API</h3>
-  <dl class="dummy-component-props">
+  <p class="dummy-paragraph">The
+    <code class="dummy-code">Dropdown</code>
+    component is composed by different parts, with their own APIs:</p>
+  <ul>
+    <li class="dummy-paragraph">
+      a main "container" (the dropdown itself)
+    </li>
+    <li class="dummy-paragraph">
+      <p>two "toggle" child components (the button that opens/closes the dropdown)</p>
+      <ul>
+        <li class="dummy-paragraph">
+          ToggleButtom
+        </li>
+        <li class="dummy-paragraph">
+          ToggleIcon
+        </li>
+      </ul>
+    </li>
+    <li class="dummy-paragraph">
+      <p>different "list-item" child components (they have different specialized scopes)</p>
+      <ul>
+        <li class="dummy-paragraph">
+          Description
+        </li>
+        <li class="dummy-paragraph">
+          Generic
+        </li>
+        <li class="dummy-paragraph">
+          Interactive
+        </li>
+        <li class="dummy-paragraph">
+          Separator
+        </li>
+        <li class="dummy-paragraph">
+          Title
+        </li>
+      </ul>
+    </li>
+  </ul>
+  <p class="dummy-paragraph"><em>
+      Notice: to make the invocation more intuitive for developers, all the sub-components are named yields, so the
+      yielded name is a simplified version of the full component name (eg.
+      <code class="dummy-code">Hds::ListItem::Interactive</code>
+      becomes just
+      <code class="dummy-code">Interactive</code>). See below how they are invoked as yielded components.
+    </em>
+  </p>
```

Comment:
[issue] This paragraph is not technically accurate. 
[issue] I also can't quite tell what you are trying to say here, so I can't make a suggestion just yet as to what it should be changed to. Let's workshop this a little.

Reply:
LOL, you wrote it :) 
https://github.com/hashicorp/design-system/commit/c048f0ebd2cdd734e72dc592b92691c690ed770e

OK to workshop this, I'll set up a meeting.

Reply:
haha it happens! Good idea, thanks!

Reply:
reworded with `is yielded in a hash under the key` and `contextual component`